.github/workflows/e2e-tests.yml

```diff
@@ -154,6 +154,13 @@ jobs:
       - name: "Uncompress build"
         run: tar -xf build-${{ matrix.os }}.tar.gz
 
+      - name: Choose latest next.js version
+        if: ${{ !startsWith(matrix.app, 'prev') }}
+        working-directory: packages/e2e-tests/${{ matrix.app }}
+        run: |
+          cd packages/e2e-tests/$APP_NAME
```

Comment:
probably not needed if you already set the working directory?

Reply:
Yeah, left over from my first try, fixed.